ovs/v3/OVS_v3.0.1.yaml

```diff
@@ -323,6 +341,12 @@ components:
             - $ref: '#/components/schemas/UNLocationLocation'
             - $ref: '#/components/schemas/FacilitySMDGLocation'
             - $ref: '#/components/schemas/AddressLocation'
+          discriminator:
```

Comment:
Hi @palatsangeetha , why is this a change if already present in 3.0.0? 
<img width="776" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/109235796/44fb13fb-a6a0-4cd1-ac17-8cddd2ed6af9">